src/shim/sdk/tests/GroupSDK.test.ts

```diff
@@ -123,16 +132,48 @@ describe("GroupSDK", () => {
                             message: {
                                 groupID: "",
                                 groupName: "",
+                                ownerUserId: "",
                                 addAsMember: false,
+                                addAsAdmin: true,
                                 needsRotation: false,
-                                memberList: [],
+                                userLists: {
+                                    memberList: [],
+                                    adminList: [],
+                                },
+                            },
+                        });
+                        done();
+                    })
+                    .catch((e) => fail(e));
+            });
+            it("fails validateOwnership if addAsAdmin id provided as false and no owner is specified", () => {
+                ShimUtils.setSDKInitialized();
+                expect(() => GroupSDK.create({addAsAdmin: false})).toThrow();
```

Comment:
Oh, I see that you're testing that method here, so no need to write another unit test for it probably.

Reply:
This test should have been in ShimUtils so I moved it there